Frontend/ui-driver/src/Flow.purs

```diff
@@ -252,6 +254,29 @@ onBoardingFlow = do
       else if (resp.rcVerificationStatus == "NO_DOC_AVAILABLE") then addVehicleDetailsflow
         else applicationSubmittedFlow "StatusScreen"
 
+updateDriverVersion :: Maybe Version -> Maybe Version -> FlowBT String Unit
+updateDriverVersion dbClientVersion dbBundleVersion = do
+  let versionName = getValueToLocalStore VERSION_NAME
+      bundle = getValueToLocalStore BUNDLE_VERSION
+      clientVersionArray = (split (Pattern ".") versionName)
+      bundleVersionArray = (split (Pattern ".") bundle)
+      (Version clientVersion) = Version {
+        major : fromMaybe 1 (fromString $ fromMaybe "NA" $ clientVersionArray !! 0),
+        minor : fromMaybe 1 (fromString $ fromMaybe "NA" $ clientVersionArray !! 1),
+        maintenance : fromMaybe 1 (fromString $ fromMaybe "NA" $ clientVersionArray !! 2)
```

Comment:
if values in local storage is not present in such case you should not call api to update with `1.1.1`